thoth/glyph/cli.py

```diff
@@ -93,18 +103,107 @@ def generate(output: str) -> None:
     "--message",
     "-m",
     type=str,
-    default="-",
+    required=True,
     help="Commit message to be classified",
 )
 def classify(message: str) -> None:
     """Generate CHANGELOG entries from the current Git project."""
     _LOGGER.info("Hello, glyph!")
-    if message == "-":
-        click.echo("Please enter message")
-    else:
-        classifier = load_model("../../data/model_commits_v2_quant.bin") 
-        label = classifier.predict(message)
-        click.echo("Label : " + str(label[0][0])[8:])
+    _LOGGER.info("Model Path : " + DEFAULT_MODEL_PATH)
+    classifier = load_model(DEFAULT_MODEL_PATH) 
+    label = classifier.predict(message.lower())
+    click.echo("Label : " + str(label[0][0])[8:])
 
+@cli.command()
+@click.option(
+    "--path",
+    "-p",
+    type=str,
+    required=True,
+    help="Path to Git repository"
+)
+@click.option(
+    "--start",
+    type=str,
+    help="Starting date"
+)
+@click.option(
+    "--end",
+    type=str,
+    help="End date"
+)
+@click.option(
+    "--start_tag",
+    type=str,
+    help="Starting date"
+)
+@click.option(
+    "--end_tag",
+    type=str,
+    help="End date"
+)
+@click.option(
+    "--output",
+    type=str,
+    help="Generated output file"
+)
+@click.option(
+    "--model",
+    type=str,
+    help="Generated output file"
+)
+def classifyrepo(path: str, start: str, end: str, start_tag: str, end_tag:str, output: str, model:str) -> None:
+    if model is None:
+        model = DEFAULT_MODEL_PATH
+    _LOGGER.info("Model Path : " + model)
+    classifier = load_model(model)
+    start_time = 0
+    end_time = sys.maxsize
+
+    if start is not None:
+        start_time = int(time.mktime(datetime.datetime.strptime(start, "%Y-%m-%d").timetuple()))
+
+    if end is not None:
+        end_time = int(time.mktime(datetime.datetime.strptime(end, "%Y-%m-%d").timetuple()))
+
+    repo = Repository(os.path.join(path, ".git"))
+    orig_messages = classify_by_date(repo, start_time, end_time)
+
+    if start_tag is not None and end_tag is not None:
```

Comment:
Does it mean the classification will be done always by date (and subsequently by tag)?

Might be good idea to introduce two sub-commands instead of putting this logic to one `classifyrepo` command.

Reply:
Sure, will create a separate command for the tag functionality

Reply:
Separate "classify-repo-by-tag" command has been added.